src/components/editor/ResourceTemplate.jsx

```diff
@@ -33,6 +34,7 @@ class ResourceTemplate extends Component {
         <div id="resourceTemplate" style={{ marginTop: '-30px' }}>
           <section>
             <h1><em>{this.props.resourceTemplate.resourceLabel}</em></h1>
+            <CopyToNewMessage />
```

Comment:
I wonder if we only want this to show up for a "saved" (has a URI) resource.

Reply:
I was debating to only enable copy with "saved" (has a URI) but I thought that a user may have filled out a Resource Template first and then would want to do a copy of their work. @michelleif do you have any opinions on this? I'm happy to change the Copy to only be enabled if the source resource has a URI.

Reply:
Isn't copying an unsaved resource just going to be a no-op?  How could the original source data be preserved if it wasn't first saved?

Reply:
I'm in agreement as well -- The link should be disabled for an unsaved resource.

Reply:
👍

Reply:
Done, commit push up to only enable the Copy button when a resource has an existing URI.